symfony-cmf/seo-bundle/2.0/config/packages/cmf_seo_bundle.yaml

```diff
@@ -0,0 +1,6 @@
+# Read the documentation: https://symfony.com/doc/master/cmf/bundles/seo/configuration.html
+cmf_seo:
+    persistence:
+        phpcr: ~
+    title: '%%content_title%% | Insert your base Title'
+    description: 'Insert you base description here. %%content_description%%'
```

Comment:
should the base description not go to the end rather? this way round, the preview of every page will start with the same text, which seems less than ideal to me.

Reply:
@dbu  We do have that configuration in sandbox and our website.